bc/core/utils/status/templates.py

```diff
@@ -104,3 +109,43 @@
 
 #CL{docket_id}""",
 )
+
+
+BLUESKY_FOLLOW_A_NEW_CASE = TwitterTemplate(
+    link_placeholders=[],
+    str_template="""I'm now following {docket}:
+
+{docket_link}
+
+#CL{docket_id}""",
+)
+
+BLUESKY_FOLLOW_A_NEW_CASE_W_ARTICLE = TwitterTemplate(
+    link_placeholders=["article_url"],
+    str_template="""I'm now following {docket}:
+
+Docket: {docket_link}
+
+Context: {article_url}
+
+#CL{docket_id}""",
+)
```

Comment:
Did you play with Markdown? 

I could see this being pretty cool with something like:

```suggestion
BLUESKY_FOLLOW_A_NEW_CASE_W_ARTICLE = TwitterTemplate(
    link_placeholders=["article_url"],
    str_template="""I'm now following {docket}:

[Docket]({docket_link}) | [Article Link]({article_url})

#CL{docket_id}""",
)
```

I'll put suggestions for others below too...